www/serf1/Portfile

```diff
@@ -50,6 +50,13 @@ variant universal {}
 build.cmd       ${prefix}/bin/scons
 
 platform darwin {
+    # Build system fails to pass Macports archflags.
+    if {${os.major} == 10 && ${configure.build_arch} eq "ppc"} {
+        # This is a fix for Rosetta build, where a physical CPU is Intel.
+        post-patch {
+            reinplace "s|@ARCH@|${configure.build_arch}|" ${worksrcpath}/SConstruct
```

Comment:
The build now fails during compilation, due to `@ARCH@` not being replaced for non-Rosetta builds.

So should the patch only be applied for Rosetta?

Reply:
Fair enough, sorry. Will fix now.